spec/spec_helper.rb

```diff
@@ -116,6 +137,23 @@ def check_for_confirmation_link
   url
 end
 
+def check_for_password_reset_link
```

Comment:
should we start putting this in separate helper method files? `spec_helper` is getting large quite fast!